### PR TITLE
Bug 1449298 - Improve all fixtures

### PR DESCRIPTION
**Pull request by MichaelKohler**

### CODE REVIEW

File: remo/profiles/fixtures/demo_users.json [hunk 1]
```diff
@@ -1,458 +1,792 @@
...
+                4,
+                6,
+                7,
+                8
             ],
             "user_permissions": [],
-            "password": "md5$2$f74916608ad95265f58a739d6ffba15f",
-            "email": "counselor@example.com",
-            "date_joined": "2012-01-23 05:30:50+00:00"
+            "password": "bcrypt2015_12_15$2b$12$u.t2bc5XxAMQfjGhGwrUpuvouOileoVab/qIkz2QLUUCsMDfI8xku$2015-12-15",
```
Comment (akatsoulas) [inline remo/profiles/fixtures/demo_users.json]: Why are you adding the password field?

File: remo/profiles/fixtures/demo_functional_areas.json [hunk 2]
```diff
@@ -17,15 +17,6 @@
             "slug": "addons"
         }
     },
-    {
```
Comment (akatsoulas) [inline remo/profiles/fixtures/demo_functional_areas.json]: Can you please order these in an ascending order by `prk`? Can you also change those in order to be consecutive?

File: remo/profiles/fixtures/demo_users.json [hunk 3]
```diff
@@ -1,458 +1,792 @@
...
-            "first_name": "Menis",
-            "last_name": "Denis",
+            "username": "peer",
+            "first_name": "Henrik",
+            "last_name": "Mitsch",
             "is_active": true,
             "is_superuser": false,
-            "is_staff": false,
-            "last_login": "2012-01-23 05:31:21+00:00",
+            "is_staff": true,
```
Comment (akatsoulas) [inline remo/profiles/fixtures/demo_users.json]: No need to add `is_stuff` to fixtures

**Comment by MichaelKohler** [comment]
@akatsoulas everything fixed, please review again :)

**Comment by MichaelKohler** [comment]
@akatsoulas this is waiting for your final review :) Thanks!

**Comment by MichaelKohler** [comment]
This actually overwrites the superuser created before loading the fixtures. Will see what I can do :)

**Comment by MichaelKohler** [comment]
Problem fixed, @akatsoulas r?

**Comment by MichaelKohler** [comment]
As discussed, I'm closing this one.

**Pull request closed without merging**